upgrade/1.2.6/scripts/update-customization.sh

```diff
@@ -33,5 +33,8 @@ kubectl -n loftsman get secret site-init -o jsonpath='{.data.customizations\.yam
 cp "${CUSTOMIZATIONS_YAML}" "${CUSTOMIZATIONS_YAML}.bak"
 yq w -i --style=single "${CUSTOMIZATIONS_YAML}" spec.kubernetes.services.cray-nls.externalHostname 'cmn.{{ network.dns.external }}'
 yq w -i --style=single "${CUSTOMIZATIONS_YAML}" spec.proxiedWebAppExternalHostnames.customerManagement[+] 'argo.cmn.{{ network.dns.external }}'
+
+# rename customazations file so k8s secret name stays the same
+mv "${CUSTOMIZATIONS_YAML}" customizations.yaml
```

Comment:
This way we'll still not overwrite the edited copy if we run a second time. 
```suggestion
cp "${CUSTOMIZATIONS_YAML}" customizations.yaml
```